### PR TITLE
ENH: Add GAN-based WM complexity identification paper review

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add GAN-based WM complexity identification paper review.

### CODE REVIEW

File: article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Authors propose a measure that considers the complexity of a voxel in its local
+context (region) to quantify the localized wiring complexity of the white
+matter. According to them, this would allow, for example, to identify
+"particularly ambiguous regions of the brain for tractographic approaches".
+
+Authors define the localized wiring complexity of white matter as how
+accurately and confidently a well-trained model can predict the missing patch.
+
+Authors seem to translate the wiring complexity as a "uncertainty along major
```
Comment (AntoineTheb) [inline article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md]: as ~a~ "uncertainty

File: article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+patterns."
+
+
+# Comments
+
+- It is unclear the relationship of their problem to image inpainting methods
+that authors mention.
+- It is unclear what the input to their network is.
+- It is unclear how they built their ground-truth.
+- MDA in not a feature that is used commonly in dMRI analysis, at least not
```
Comment (AntoineTheb) [inline article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md]: MDA _is_ not

File: article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+under that denomination.
+- The paper lacks a better rationale and explanation about their model/network
+architecture.
+- No quantitative results are presented.
+- No comparisons against other methods are presented to allow users assess the
```
Comment (AntoineTheb) [inline article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md]: users _to_ assess

File: article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md (hunk 1 above)
Comment (jhlegarreta) [reply article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md]: Done in 296cb4b. Thanks !

File: article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md (hunk 2 above)
Comment (jhlegarreta) [reply article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md]: Fixed in 296cb4b. Thanks !

File: article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md (hunk 3 above)
Comment (jhlegarreta) [reply article/_posts/2019-11-21-GANBasedWMComplexityIdentification.md]: Fixed in 296cb4b. Thanks !

**Pull request merged**